src/charm.py

```diff
@@ -65,6 +70,9 @@
 CA_CERTIFICATE_JUJU_SECRET_LABEL = "vault-ca-certificate"
 SEND_CA_CERT_RELATION_NAME = "send-ca-cert"
 VAULT_INITIALIZATION_SECRET_LABEL = "vault-initialization"
+S3_RELATION_NAME = "s3-parameters"
+DEFAULT_REGION = "us-east-1"  # It is used as default in boto3
```

Comment:
We have this set here as well as in the s3 file, we should probably only keep one to avoid having to guess which one is the right one.

Reply:
Now it's only handled in the s3 file.